tests/main/security-devpts/task.yaml

```diff
@@ -1,5 +1,10 @@
 summary: Ensure that the basic devpts security rules are in place.
 
+details: |
+    Test that a snap which has a plug for the physical-memory-observe interface
+    can read /dev/pts, can openpty, and can access the slave PTY via /dev/ptmx,
```

Comment:
we need to avoid the 'slave' word

Reply:
I thought this was the (unfortunate) technical term, found in `man pty` and `man openpty`. Do you have a suggestion for what to call it instead?

Reply:
Perhaps I can just `s/slave //`, since someone who understands PTY knows that it's the slave PTY which is found at `/dev/ptmx`